recipes-debian/xorg-app/xkbcomp_debian.bb

```diff
@@ -0,0 +1,37 @@
+# base recipe: meta/recipes-graphics/xorg-app/xkbcomp_1.4.2.bb
+# base branch: warrior
+# base commit: 4623554463685000907eb3e17b4b23136f3759a2
+
+SUMMARY = "A program to compile XKB keyboard description"
+
+DESCRIPTION = "The xkbcomp keymap compiler converts a description of an \
+XKB keymap into one of several output formats. The most common use for \
+xkbcomp is to create a compiled keymap file (.xkm extension) which can \
+be read directly by XKB-capable X servers or utilities."
+
+HOMEPAGE = "http://www.x.org/"
+BUGTRACKER = "https://bugs.freedesktop.org"
+SECTION = "x11/apps"
+LICENSE = "MIT-X"
+DEPENDS = "util-macros-native virtual/libx11"
+
+# depends on virtual/libx11
+REQUIRED_DISTRO_FEATURES = "x11"
+
+inherit debian-package
+require recipes-debian/sources/x11-xkb-utils.inc
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/x11-xkb-utils.git"
+S = "${DEBIAN_UNPACK_DIR}/xkbcomp"
+PV = "1.4.0"
```

Comment:
@yuwata  Do we need change PV from 7.7+4(is defined in recipes-debian/sources/x11-xkb-utils.inc) to 1.4.0?

Reply:
Yes. I will add a comment about that. Thanks.

Reply:
A comment is added. PTAL.

Reply:
thanks!